api/EXT_sRGB.json

```diff
@@ -5,40 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/EXT_sRGB",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "40"
           },
           "chrome_android": {
-            "version_added": true
+            "version_added": "40"
           },
           "edge": {
-            "version_added": "79"
+            "version_added": false
```

Comment:
This also seems suspect. A testing problem?

Reply:
This is one of the extensions Edge doesn't support.